benchmarks/full_report/vbench/experiment.py

```diff
@@ -0,0 +1,49 @@
+#  Copyright (c) 2024. Lance Developers
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""This file contains the experiment configuration for the benchmarking
+framework. The configuration is loaded from a yaml file and converted
+into a Benchmark object."""
+
+from typing import Dict
+import attrs
+import lance
+import numpy as np
+
+
+def _load_npy(path: str) -> np.ndarray:
+    return np.load(path)
+
+
+@attrs.define
+class AnnTest:
+    name: str = attrs.field()
+    query: np.ndarray = attrs.field(converter=_load_npy)
+    ground_truth: np.ndarray = attrs.field(converter=_load_npy)
```

Comment:
there a reason not to just pass the functions as is?

```suggestion
    query: np.ndarray = attrs.field(converter=np.load)
    ground_truth: np.ndarray = attrs.field(converter=np.load)
```